.github/workflows/gcr-cleaner.yaml

```diff
@@ -0,0 +1,32 @@
+# .github/workflows/gcr-cleaner.yaml
+name: 'gcr-cleaner'
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+jobs:
+  gcr-cleaner:
+    runs-on: 'ubuntu-latest'
+    steps:
+      # configure based on registry
+      - uses: 'docker/login-action@v2'
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
```

Comment:
The secret we already have for DockerHub login is `DOCKERHUB_ACCESS_TOKEN` and we've created a GitHub-specific access token so that we can easily revoke it as necessary. Did you add a `DOCKERHUB_PASSWORD` to the Actions secrets? If so could we delete it and use the access token instead? That will make managing and revoking things much easier. 

https://github.com/trisacrypto/directory/blob/main/.github/workflows/containers.yaml#L58

Reply:
will update this. did not actually add the `DOCKERHUB_PASSWORD`